app/src/main/java/org/mozilla/tv/firefox/pocket/PocketVideoFetchScheduler.kt

```diff
@@ -110,6 +111,11 @@ class PocketVideoFetchScheduler(
         return userFetchTime.timeInMillis - now.timeInMillis
     }
 
+    fun setDelayForQA(seconds: Long) {
```

Comment:
nit: -> setQAFetchDelayOverride ?

I think "override" indicates that this will be influenced by an outside source while `setDelayForQA` could just be an internal method. I changed the word order to be consistent with the command line flag.

This function should be similarly named to the function in IntentValidator, imo.